specs/commandline/blob.md

```diff
@@ -0,0 +1,362 @@
+# notation policy
+
+## Description
+
+Use `notation blob` command to sign, verify, and inspect signatures associated with arbitrary blobs. Notation can sign and verify any arbitrary bag of bits like zip files, documents, executables, etc. When a user signs a blob, `notation` produces a detached signature, which the user can transport/distribute in any medium that the user prefers along with the original blob. On the verification side, Notation can verify the blob's signature and assert that the blob has not been tampered with during its transmission. For more details, see [trust policy specification and examples](https://github.com/notaryproject/specifications/blob/main/specs/signing-and-verification-workflow.md#blob-signing-workflow).
+
+## Outline
+
+### notation blob command
+
+```text
+Sign, Inspect, and Verify signatures associates with arbitrary blobs.
+
+Usage:
+  notation blob [command]
+
+Available Commands:
+  sign      produce a detached signature for a given blob
+  inspect   inspect a signature associated with a blob
+  verify    verify a signature associated with a blob
+
+Flags:
+  -h, --help   help for blob
+```
+
+### notation blob sign
+
+```text
+Produce a detached signature for a given blob
+
+Usage:
+  notation blob sign [flags] <blob_path>
+
+Flags:
+  -s,  --signature path             output location of the detached signature
```

Comment:
Is this a required flag? If it is, we might need to update the usage schema. If it's not a required flag, what's the default behavior?

Reply:
It is a required flag. I will updated the schema.